docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **boundaries**: Boundary `name:left` and `name:right` values sometimes flipped, especially for `country` and `region` features (Issue [#1770](https://github.com/tilezen/vector-datasource/issues/1770))
+  * **boundaries**: Made `min_zoom` of `country` lines depend on the data.
+  * **roads**: Fix v1.6 regression where `runway` and `taxiway` lines were sorted below **landuse** layer polygons for the same in error, thanks @bcamper! (Issue [#1814](https://github.com/tilezen/vector-datasource/issues/1814))
+  * **roads**: Expand `is_bridge` to include viaduct and any other not "no" bridges (Issue [#1314](https://github.com/tilezen/vector-datasource/issues/1314))
+  * **earth**: Add `min_zoom` properties. (Issue [#1073](https://github.com/tilezen/vector-datasource/issues/1073))
+  * **water**: Drop all lake name variants zooms 0-4, again. (Issue [#1730](https://github.com/tilezen/vector-datasource/issues/1730))
+  * **water**: Drop additional lake names and variants at mid- and high-zooms (zooms 5-15) based on area (Issue [#1730](https://github.com/tilezen/vector-datasource/issues/1730))
+  * **water**: Update min zoom of water label points based on their inclusion in tiles not just their raw NE data value.
+  * **water**: 0/0/0 tile has clipped water content (Issue [#1806](https://github.com/tilezen/vector-datasource/issues/1806))
+  * **water**: Extract water boundaries at zoom 8, with the switch to OSM data from NE.
+  * **places**: Don't emit `area=0` on point labels (Issue [#1825](https://github.com/tilezen/vector-datasource/issues/1825))
+  * **places**: Default `country` labels to **zoom 6** instead of 1 when there isn't Natural Earth match (Issue [#1826](https://github.com/tilezen/vector-datasource/issues/1826))
+  * **places**: Default `region` labels to **zoom 8** instead of 1 when there isn't Natural Earth match (Issue [#1826](https://github.com/tilezen/vector-datasource/issues/1826))
+
+  #### DOCUMENTATION CHANGES
+
+    TODO update JSON
+  * Updated Layers documentation for v1.7 schema changes.
+  * Publish docs to [tilezen.readthedocs.io](https://tilezen.readthedocs.io).
+  * Update documentation formatting for ReadTheDocs.io Markdown requirements.
+  * Restructure some file layout for ReadTheDocs.io build system, including: `CHANGELOG`, `LICENSE`, `CONTRIBUTING`, `MIGRATION_GUIDE`, `PERFORMANCE`, `SEMANTIC-VERSIONING`, and `TEST` files.
+  * Corrected order of `low_emission_zone` in list.
+  * Generalized `hgv_restriction` property units to mostly meters for heavy goods vehicle truck access restrictions.
+
+  #### INTERNAL CHANGES
+
+  * Add `gunicorn` to dependencies, thanks [@rwrx](https://github.com/rwrx). [PR #1690](https://github.com/tilezen/vector-datasource/pull/1690)
+  * Bump `PyYAML` version for CVE-2017-18342.
+  * Stop using `tags-> in YAML (Issue [#1199](https://github.com/tilezen/vector-datasource/issues/1199))
```

Comment:
`` `tags-> `` should have a trailing `` ` ``.

Reply:
Fixed in 7a06d57557640cee0bd129734b05aeba10de3085.